test/GithubApi.Put.test.js

```diff
@@ -0,0 +1,67 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+
+const { expect } = chai;
+
+describe('following a user on github', () => {
+  let following;
+  let followingStatus;
+
+  before(() => agent.put('https://api.github.com/user/following/aperdomob')
+    .auth('token', process.env.ACCESS_TOKEN)
+    .then((response) => {
+      following = response;
+      followingStatus = response.status;
+    }));
+
+  it('Then aperdomob should be followed', () => {
+    expect(followingStatus).to.equal(statusCode.NO_CONTENT);
+    expect(following.body).to.eql({});
+  });
+
+  describe('finding a followed user', () => {
+    let followedUsers;
+
+    before(() => agent.get('https://api.github.com/user/following')
+      .auth('token', process.env.ACCESS_TOKEN)
+      .then((response) => {
+        followedUsers = response.body;
+      }));
+
+    it('then user is alrready followed', () => {
+      expect(followedUsers).to.containSubset([{ login: 'aperdomob' }]);
+    });
+
+    describe('checkind "idempotencia"', () => {
+      let followingRevisited;
+      let followingStatusRevisited;
+
+      before(() => agent.put('https://api.github.com/user/following/aperdomob')
+        .auth('token', process.env.ACCESS_TOKEN)
+        .then((response) => {
+          followingRevisited = response;
+          followingStatusRevisited = response.status;
+        }));
+
+      it('Then aperdomob should be followed', () => {
```

Comment:
tienes dos it iguales, para diferenciarlos puedes poner algo como `aperdomob should remain followed and no error should be returned`